js/pages/characterizations/services/conversion/PrevalenceStatConverter.js

```diff
@@ -29,7 +29,16 @@ define([
 		}
 
 		getDefaultColumns(analysis) {
-		    return [ this.getСovNameColumn() ];
+		    return [
+		        this.getСovNameColumn(),
+                {
+                    title: 'Concept ID',
+                    data: 'conceptId',
+                    render: (d, t, r) => {
+                        return `<a href="/#/concept/${r.conceptId}" data-bind="tooltip: '${r.conceptName}'">${r.conceptId}</a>`
```

Comment:
While testing this, I ran into an issue caused by a concept name with an apostrophe in it - perhaps use `CommonUtils.renderBoundLink`?